app/src/test/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegrationTest.kt

```diff
@@ -107,6 +108,24 @@ class BrowserToolbarIntegrationTest {
         verify(browserToolbarIntegration).observerSecurityIndicatorChanges()
     }
 
+    @Test
+    fun `WHEN start method is called THEN observe erase tabs CFR changes`() {
+        doNothing().`when`(browserToolbarIntegration).observeEraseCfr()
+
+        browserToolbarIntegration.observeEraseCfr()
+
+        verify(browserToolbarIntegration).observeEraseCfr()
+    }
+
+    @Test
+    fun `WHEN stop method is called THEN erase tabs scope is not active anymore`() {
+        browserToolbarIntegration.observeEraseCfr()
+
+        browserToolbarIntegration.stop()
+
+        assertEquals(browserToolbarIntegration.eraseTabsCfrScope?.isActive, false)
```

Comment:
This way of testing a loose indication that we listen for "erase tabs cfrs updates" could also be used for `browserToolbarIntegration.start()` where we could also test the feature flag and the different behavior depending on if the current tab is a custom one or not.

Reply:
After coroutines version update this test is failing but I'm not sure why.
In order to not block this merge will create a separate ticket to increase the coverage of this class